addons/@mlabs-haskell/godot-cardano/src/bool_data.gd

```diff
@@ -1,4 +1,8 @@
 extends RefCounted
+## The boolean representation used in Plutus
+##
+## This is a utility class used in the serialization of [PlutusData]. Do not
+## use it directly.
```

Comment:
Good point, I ended up removing that sentence. I think it's fine to have a separate class for this type because it makes the API consistent (why are there separate classes for things like big integers and sum types but not booleans?).